### ISSUE TITLE
文字编辑

### ISSUE THREAD

**Issue by liuyifeng593** [tyxsspa/AnyText#64]
图片文字编辑现在不好用，只有示例里的文字编辑能达到期望的效果，自己上传图片，要编辑里面的文字，一般达不到想要的效果
![82B9854834A2B26F10C16EC70761CB72](https://github.com/tyxsspa/AnyText/assets/155711572/c5dfe255-a160-4bb4-ac3a-93474a885cef)

**Comment by Ruby-He** [comment]
> 图片文字编辑现在不好用，只有示例里的文字编辑能达到期望的效果，自己上传图片，要编辑里面的文字，一般达不到想要的效果 ![82B9854834A2B26F10C16EC70761CB72](https://private-user-images.githubusercontent.com/155711572/300711775-c5dfe255-a160-4bb4-ac3a-93474a885cef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU2NzE4MTYsIm5iZiI6MTcxNTY3MTUxNiwicGF0aCI6Ii8xNTU3MTE1NzIvMzAwNzExNzc1LWM1ZGZlMjU1LWExNjAtNGJiNC1hYzNhLTkzNDc0YTg4NWNlZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxNFQwNzI1MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wOWZhNmRhMWQyMmE3NzQxODVmNGVjODRhODE2NzBlNTc2YjhhZDdlMWQ5MWY5OTk3MzViY2VkZjQ4NDEwYmUyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.QILn3Rqeu-IBQKIrp_ict6f1tu6iRXhBom_P1nmPyqI)

Indeed, if you input the edited content as "" or "*", some hallucination also occurs.
